package.json

```diff
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "test": "make test && make test-chrome-headless",
-    "lint": "eslint .",
+    "lint": "eslint . && eslint --ext md documentation",
```

Comment:
Unfortunately the `--ext` switch is the only way to specify file extensions that eslint should process besides `.js`, so we can't move this to `.eslintrc.js` or upstream it to `eslint-plugin-markdown`.